.bitmap

```diff
@@ -11,253 +11,259 @@
 {
     "accordion": {
         "scope": "showtime.universal",
-        "version": "0.0.7",
+        "version": "0.0.11",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/accordion"
     },
     "activity-indicator": {
         "scope": "showtime.universal",
-        "version": "0.0.6",
+        "version": "0.0.10",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/activity-indicator"
     },
     "alert": {
         "scope": "showtime.universal",
-        "version": "0.0.8",
+        "version": "0.0.13",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/alert"
     },
     "avatar": {
         "scope": "showtime.universal",
-        "version": "0.0.3",
+        "version": "0.0.7",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/avatar"
     },
     "bottom-sheet": {
         "scope": "showtime.universal",
-        "version": "0.0.7",
+        "version": "0.0.12",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/bottom-sheet"
     },
     "button": {
         "scope": "showtime.universal",
-        "version": "0.0.7",
+        "version": "0.0.12",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/button"
     },
     "checkbox": {
         "scope": "showtime.universal",
-        "version": "0.0.8",
+        "version": "0.0.12",
         "mainFile": "index.tsx",
         "rootDir": "packages/design-system/checkbox"
     },
+    "color-scheme": {
+        "scope": "",
+        "version": "",
```

Comment:
Looks like something missing here

Reply:
since it hasn't been released yet, trying.

Reply:
yeah i had renamed @showtime-xyz/universal.color-scheme-provider to @showtime-xyz/universal.color-scheme
sorry. naming OCD kicked in

Reply:
> naming OCD kicked in

🤣